CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Fixed a number of issues related to accessing the GUI from outside the EDT
 - Added a few missing translation strings
 - Fixed NullPointerException when opening Customize entry type dialog without an open database
+- Fixed localization error when changing key bindings while not using the english localization
```

Comment:
Instead of "Fixed localization error when changing key bindings ..." I would rather write "Fixed modified keybindings not working ..."